Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SplashView.swift
+//  Survey
+//
+//  Created by Khanh on 22/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+struct SplashView: View {
+
+    private var input: SplashViewModel.Input
+    @ObservedObject var output: SplashViewModel.Output
+
+    private let cancelBag = CancelBag()
```

Comment:
Please remove this one as well 🙏

Reply:
I removed it here : https://github.com/nkhanh44/ic-survey-ios/pull/7/commits/d05f1646f773bfbffcc774fb2425322acb897c06